### ISSUE TITLE
The Rich text labels are deleted with every change to the slice when saving from the slicemachine UI

### ISSUE THREAD

**Issue by renardsas** [prismicio/slice-machine#1100]
### Is your feature request related to a problem? Please describe.

As specified in the documentation, one can add labels to the richtext to apply specific styles :
[rich-text](https://prismic.io/docs/rich-text#custom-labels)

Exemple : 
```json
"text": {
    "type": "StructuredText",
    "config": {
      "label": "Text",
      "placeholder": "",
      "allowTargetBlank": true,
      "multi": "paragraph,preformatted,heading2,heading3,heading4,heading5,heading6,strong,em,hyperlink,list-item,o-list-item",
     "labels": ["text-blue"]
    }
  }
```

When modifying the field from slicemachine UI upon saving, it deletes the 'labels' property from the JSON.
```json
"text": {
    "type": "StructuredText",
    "config": {
      "label": "Text",
      "placeholder": "",
      "allowTargetBlank": true,
      "multi": "paragraph,preformatted,heading2,heading3,heading4,heading5,heading6,strong,em,hyperlink,list-item,o-list-item"
    }
  }
```

### Describe the solution you'd like
This is not a bug, as it's normal for the field added directly from the JSON file to disappear upon saving.
Perhaps add a field in the slicemachine UI to allow for the addition of labels ?

**Comment by nmastroianni** [comment]
I am also confirming this as a likely "bug" or as is suggested, a missing piece to the slicemachine UI?  When I add labels to the model.json file, it requires a save in SliceMachine before simulating the slice again. If you save the slice, it removes your custom labels. 😢The labels are a key design piece for editors. I look forward to a resolution of this one. Thank you @renardsas for logging this issue.

**Comment by comeprismic** [comment]
Hi, thanks for submitting the issue. We are aware of it and plan to work on a fix over the next cycles. I'll come back to you when a fix is released.

**Comment by comeprismic** [comment]
The bug was fixed in [v1.19.0](https://github.com/prismicio/slice-machine/releases). I close the issue.

**Comment by renardsas** [comment]
Thanks ! 🙏🙏🙏
